app/src/main/java/io/github/hidroh/materialistic/CustomTabsDelegate.java

```diff
@@ -110,8 +112,36 @@ void onServiceDisconnected() {
     }
 
     private static String getPackageNameToUse(Context context) {
-        // packagesSupportingCustomTabs contains all apps that can handle both VIEW intents
-        // and service calls.
+        List<String> browserWithCustomTabsSupport = getBrowserWithCustomTabsSupport(context);
+        String defaultBrowser = getDefaultBrowser(context);
+
+        for (String browser : browserWithCustomTabsSupport) {
+            if (browser.equals(defaultBrowser)) {
```

Comment:
Can you use `TextUtils.equals(browser, defaultBrowser)` instead? It's a null-safe version of String comparison since it is possible that `defaultBrowser` would be null.

Reply:
Done